### PR TITLE
Adding Plausible Analytics script 

### PR DESCRIPTION
**Pull request by HarleyBurrowTPX**
- Added Plausible analytics script

### CODE REVIEW

File: app/views/shared/_head.html.erb [hunk 1]
```diff
@@ -6,4 +6,5 @@
   <%= csp_meta_tag %>
   <%= stylesheet_link_tag "application" %>
   <%= javascript_include_tag "application" %>
+  <script defer data-domain="data-catalogue-prototypes-main.herokuapp.com/manage-prototype" src="https://plausible.io/js/script.js"></script>
```
Comment (em-cd) [inline app/views/shared/_head.html.erb:9]: The domain here is referring to Lewis' prototypes, should it be instead the domain for whichever environment it's in? Presumably we'll only want Plausible running in the environment we use for production?

**Review by em-cd — CHANGES_REQUESTED** [review]

File: app/views/shared/_plausible_script.html.erb [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<% if ENV["PLAUSIBLE_DOMAIN"] && ENV["PLAUSIBLE_DOMAIN"].include?("staging") %>
```
Comment (em-cd) [inline app/views/shared/_plausible_script.html.erb:1]: Not sure why we need the second part of the check to see that the domain contains the word 'staging'? I think if the environment variable is set we can probably assume that we want the script to run.

Comment (HarleyBurrowTPX) [reply app/views/shared/_plausible_script.html.erb:1]: I wanted to account for that but forgot the variable in test but forgot it is only present in the staging variables. I will remove.

**Review by em-cd — APPROVED** [review]

File: app/views/shared/_plausible_script.html.erb [hunk 3]
```diff
@@ -0,0 +1,20 @@
+ <% if Rails.env.staging? %>
+  <!-- Only render plausible script in staging -->
+    <script defer data-domain=<%= Rails.env['PLAUSIBLE_DOMAIN'] %>  src="https://plausible.io/js/script.manual.js"></script>
```
[the remark this answers is not in the document]
Comment (ngw) [reply app/views/shared/_plausible_script.html.erb]: `ENV.fetch('PLAUSIBLE_DOMAIN')` https://stackoverflow.com/questions/47985660/rails-difference-between-env-fetch-and-env
You have to either set a default of some sort (which might just not work, for example in dev) or set the ENV var in docker and ensure it's picked up on heroku.

**Pull request merged**